src/components/blocks/exhibition-promo/_exhibition-promo.scss

```diff
@@ -22,47 +42,6 @@
     }
   }
 
```

Comment:
I think i see why you re-ordered this section, to group all the modifying styles together at the end as someone had started doing previously, but you've left one stray.  
However, all of the modifying rules should be located immediately after the rules they modify, for clarity when developing them further. The previous structure wasn't good. Could you return it to the original order please, (which generally should match the order that the classes descend in the HTML), but then move each modifying rule to sit under its original rule, like so:
```
&__title {...}
&--double &__title {...}
```
etc.

Reply:
Thanks, as requested, "all of the modifying rules should be located immediately after the rules they modify, for clarity when developing them further" - these rules have now been rearranged.